medmodels/medrecord/types.py

```diff
@@ -87,6 +87,26 @@
 #: A type alias for input to a Pandas DataFrame for edges.
 PandasEdgeDataFrameInput: TypeAlias = Tuple[pd.DataFrame, str, str]
 
+#: A type alias for input to a node.
+NodeInput = Union[
+    NodeTuple,
+    Sequence[NodeTuple],
+    PandasNodeDataFrameInput,
+    List[PandasNodeDataFrameInput],
+    PolarsNodeDataFrameInput,
+    List[PolarsNodeDataFrameInput],
+]
+
+#: A type alias for input to an edge.
+EdgeInput = Union[
+    EdgeTuple,
+    Sequence[EdgeTuple],
```

Comment:
```suggestion
    List[EdgeTuple],
```
as `EdgeTuple` is covariant

Reply:
Created new Tying EdgeInputBuilder and NodeInputBuilder in the builder.py file